contrib/devtools/README.md

```diff
@@ -7,22 +7,65 @@ clang-format.py
 
 A script to format cpp source code according to [.clang-format](../../src/.clang-format). This should only be applied to new files or files which are currently not actively developed on. Also, git subtrees are not subject to formatting.
 
-fix-copyright-headers.py
-========================
-
-Every year newly updated files need to have its copyright headers updated to reflect the current year.
-If you run this script from the root folder it will automatically update the year on the copyright header for all
-source files if these have a git commit from the current year.
-
-For example a file changed in 2015 (with 2015 being the current year):
-
-```// Copyright (c) 2009-2013 The Bitcoin Unlimited developers```
-// Copyright (c) 2015-2017 The Bitcoin Unlimited developers
-
-would be changed to:
+copyright\_header.py
+====================
 
-```// Copyright (c) 2009-2015 The Bitcoin Unlimited developers```
-// Copyright (c) 2015-2017 The Bitcoin Unlimited developers
+Provides utilities for managing copyright headers of `The Bitcoin Core
+developers` in repository source files. It has three subcommands:
+>>>>>>> 159597a2b... [devtools] script support for managing source file copyright headers
+
+```
+$ ./copyright_header.py report <base_directory> [verbose]
+$ ./copyright_header.py update <base_directory>
+$ ./copyright_header.py insert <file>
+```
+Running these subcommands without arguments displays a usage string.
+
+copyright\_header.py report \<base\_directory\> [verbose]
+---------------------------------------------------------
+
+Produces a report of all copyright header notices found inside the source files
+of a repository. Useful to quickly visualize the state of the headers.
+Specifying `verbose` will list the full filenames of files of each category.
+
+copyright\_header.py update \<base\_directory\> [verbose]
+---------------------------------------------------------
+Updates all the copyright headers of `The Bitcoin Core developers` which were
+changed in a year more recent than is listed. For example:
+```
+// Copyright (c) <firstYear>-<lastYear> The Bitcoin Core developers
+```
+will be updated to:
+```
+// Copyright (c) <firstYear>-<lastModifiedYear> The Bitcoin Core developers
+```
+where `<lastModifiedYear>` is obtained from the `git log` history.
+
+This subcommand also handles copyright headers that have only a single year. In
+those cases:
+```
+// Copyright (c) <year> The Bitcoin Core developers
+```
+will be updated to:
+```
+// Copyright (c) <year>-<lastModifiedYear> The Bitcoin Core developers
+```
```

Comment:
maybe should change reference of Bitcoin Core developers to just "Bitcoin developers"

Reply:
nvm...i see you already did this in a later commit.